src/main/scala/com/github/matsluni/akkahttpspi/AkkaHttpClient.scala

```diff
@@ -139,7 +139,9 @@ object AkkaHttpClient {
 
   case class AkkaHttpClientBuilder(private val actorSystem: Option[ActorSystem] = None,
                                    private val executionContext: Option[ExecutionContext] = None,
-                                   private val connectionPoolSettings: Option[ConnectionPoolSettings] = None) extends SdkAsyncHttpClient.Builder[AkkaHttpClientBuilder] {
+                                   private val connectionPoolSettings: Option[ConnectionPoolSettings] = None,
+                                   private val protocol: HttpProtocol = HttpProtocols.`HTTP/1.1`
+                                  ) extends SdkAsyncHttpClient.Builder[AkkaHttpClientBuilder] {
```

Comment:
It seems with your latest refactoring here, the whole `attributeMap` code is accidently gone.

Reply:
Maybe we can also use a new `withProtocol(HttpProtocol)` builder method to set the protocol?

Reply:
> It seems with your latest refactoring here, the whole attributeMap code is accidently gone.

the removal of the whole `attributeMap` code was on purpose. let's explore that in #232 

> Maybe we can also use a new `withProtocol(HttpProtocol)` builder method to set the protocol?

yes I can do that

Reply:
> > It seems with your latest refactoring here, the whole attributeMap code is accidently gone.
> 
> the removal of the whole `attributeMap` code was on purpose. let's explore that in #232

ok, but now I realize the README and tests don't make sense 🤦